requirements.txt

```diff
@@ -1,5 +1,6 @@
 geolinks==0.2.0
 lxml==3.3.5
-OWSLib<0.9.0
```

Comment:
Tests did not pass with OWSLib 0.9.x in the past.
Has something changed with that?